docs/stack/use-cases/_index.md

```diff
@@ -0,0 +1,417 @@
+
+- Wind (km/h)
+- Ocean conditions (classes)
+- Speed (knots)
+- Location (long, lat)
+
+**Connected car (sensor data and IoT)**
+
+Modern cars are exposing several metrics via a standard interface. Use TimeSeries to correlate average fuel consumption with the tire pressure, figure out how long to keep a car in the fleet, determine optimal maintenance intervals, and calculate tax savings by type of the road (taxable vs. nontaxable roads). Example metrics include:
+
+- Acceleration
+- Location (long, lat)
+- Fuel level (liter)
+- Distances (km)
+- Speed (km/h)
+- Tire pressure
+- Distance until next maintenance check
+
+**Smart metering (sensor data and IoT)**
+
+Modern houses and facilities gather details about energy consumption/production. Use RedisTimeSeries to aggregate billing based on monthly consumption. Optimize the network by redirecting the energy delivery relative to the fluctuations in need. Provide recommendations on how to improve the energy consumption behavior. Example metrics include:
+
+- Consumption per location
+- Produced amount of electrical energy per location
+
+**Quality of service (telecom)**
+
+Mobile phone usage is increasing, producing a natural growth that just correlates to the increasing number of cellphones. However, there might also be spikes that correlate with specific events (for example, more messages around world championships). 
+
+Telecom providers need to ensure that they are providing the necessary infrastructure to deliver the right quality of service. This includes using mini towers for short-term peaks. Use TimeSeries to correlate traffic peaks to specific events, load balance traffic over several towers or mini towers, and predictively plan the infrastructure. Metrics include the amount of traffic per tower.
+
+**Stock trading (finance)**
+
+Stock trading is highly automated today. Algorithms, and not just human beings, are trading, from the amount of bids and asks for the trading of a stock to the extreme volumes of trades per second (millions of ops per second). Computer-driven trading requires millisecond response times. It's necessary to keep a lot of data points within a very short period of time (for example, price fluctuations per second within a minute). In addition, the long-term history needs to be kept to make statements about trends or for regulatory purposes.  
+
+Use RedisTimeSeries to identify correlations between the trading behavior and other events (for example, social network posts). Discover a developing market. Detect anomalies to discover insider trades. Example metrics include:
+
+- Exact time and order of a trade by itself
+- Type of the event (trade/bid)
+- The stock price
+
+### Learn more
+
+[RedisTimeSeries](/docs/stack/timeseries)
+
+## RedisBloom
+
+Probabilistic data structures provide a variety of different properties that can have applications across many different industries. 
+
+### Bloom filters 
```

Comment:
Is there a reason for having separate use cases for Bloom and Cuckoo filters?

Reply:
None other than that is how they are presented in the technical enablement course for the field. What's your take?

Reply:
Bloom and Cuckoo filters basically solve the same problem.

In Redis implementation, CF has some additional functionality over BF: [CF.ADDNX](https://redis.io/commands/cf.addnx/), [CF.INSERTNX](https://redis.io/commands/cf.insertnx/), [CF.COUNT](https://redis.io/commands/cf.count/), [CF.DEL](https://redis.io/commands/cf.del/). 

I think we should walk over the examples and check if indeed these are utilized.

There are also some technical differences that are summarized nicely [here](https://brilliant.org/wiki/cuckoo-filter/)

Reply:
Joined them together!